crates/bitwarden-error-macro/src/args.rs

```diff
@@ -0,0 +1,25 @@
+use darling::FromMeta;
+
+#[derive(FromMeta)]
+pub(crate) struct BitwardenErrorArgs {
+    #[darling(flatten)]
+    pub error_type: BitwardenErrorType,
+
+    #[darling(default)]
+    pub export_as: Option<String>,
+}
+
+#[derive(FromMeta)]
+pub(crate) enum BitwardenErrorType {
```

Comment:
You should be able to use `#[darling(rename_all = "camelCase")]` once at the enum definition to avoid repeating the rename for each variant.